src/components/InitializeApp.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import { LoadingMessage } from "./LoadingMessage";
+import { setLanguageFirstTime } from "../i18n";
+
+export class InitializeApp extends React.Component<
```

Comment:
I don't think we need this component. Let's put the into `<App>`'s `initializeScene()`. This will also have the benefit of supporting parallel fetching (of scene data & language) if we want to.

Reply:
@dwelle I tried that first but for some reason, translations start to happen before the `initializeScene` even get fire. One example is [translating the title](https://github.com/excalidraw/excalidraw/blob/master/src/appState.ts#L41).

So this needs to happen before `App` even gets mounted. 

We can do some refactoring if you feel like it but it was a bit complicated for me so I went this route.